docs/book/provider_implementations/create_actuators.md

```diff
@@ -0,0 +1,160 @@
+        clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+        client "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
+)
+
+// Actuator is responsible for performing cluster reconciliation
+type Actuator struct {
+        clustersGetter client.ClustersGetter
+}
+
+// ActuatorParams holds parameter information for Actuator
+type ActuatorParams struct {
+        ClustersGetter client.ClustersGetter
+}
+
+// NewActuator creates a new Actuator
+func NewActuator(params ActuatorParams) (*Actuator, error) {
+        return &Actuator{
+                clustersGetter: params.ClustersGetter,
+        }, nil
+}
+
+// Reconcile reconciles a cluster and is invoked by the Cluster Controller
+func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
+        glog.Infof("Reconciling cluster %v.", cluster.Name)
+        return fmt.Errorf("TODO: Not yet implemented")
+}
+
+// Delete deletes a cluster and is invoked by the Cluster Controller
+func (a *Actuator) Delete(cluster *clusterv1.Cluster) error {
+        glog.Infof("Deleting cluster %v.", cluster.Name)
+        return fmt.Errorf("TODO: Not yet implemented")
+}
+```
+
+## Machine Actuator
+
+The following actuator stub code should be copied to
+`cluster-api-provider-solas/pkg/cloud/solas/actuators/machine/actuator.go`.
+
+```go
+/*
+Copyright 2018 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machine
+
+import (
+        "fmt"
+
+        "github.com/golang/glog"
+        clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+        client "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
+)
+
+const (
+        ProviderName = "solas"
+)
+
+// Actuator is responsible for performing machine reconciliation
+type Actuator struct { 
+        machinesGetter client.MachinesGetter
+}
+
+// ActuatorParams holds parameter information for Actuator
+type ActuatorParams struct {
+        MachinesGetter client.MachinesGetter
+}
+
+// NewActuator creates a new Actuator
+func NewActuator(params ActuatorParams) (*Actuator, error) {
+        return &Actuator{
+                machinesGetter: params.MachinesGetter,
+        }, nil
+}
+
+// Create creates a machine and is invoked by the Machine Controller
+func (a *Actuator) Create(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
+        glog.Infof("Creating machine %v for cluster %v.", machine.Name, cluster.Name)
```

Comment:
Using the standard lib logger instead of glog here would allow for fewer dependencies and fewer 3rd party libraries for new users to learn